common/build/eslint-config-fluid/recommended.js

```diff
@@ -139,6 +139,14 @@ module.exports = {
 				"unicorn/consistent-function-scoping": "off",
 			},
 		},
+		{
+			// Rules for test code
+			files: ["*.spec.ts", "*.test.ts", "**/test/**"],
+			rules: {
+				// Does not work well with describe/it block scoping
+				"unicorn/consistent-function-scoping": "off",
```

Comment:
An alternative option would be to disable this universally, since this is now the 2nd place we've disabled it. I don't know how valuable this rule really is,